crates/wit-component/src/cli.rs

```diff
@@ -43,19 +43,19 @@ fn parse_interface(name: Option<String>, path: &Path) -> Result<Interface> {
 #[clap(name = "component-encoder", version = env!("CARGO_PKG_VERSION"))]
 pub struct WitComponentApp {
     /// The path to an interface definition file the component imports.
-    #[clap(long = "import", value_name = "NAME=INTERFACE", parse(try_from_str = parse_named_interface))]
+    #[clap(long = "import", value_name = "NAME=INTERFACE", value_parser = parse_named_interface)]
     pub imports: Vec<Interface>,
 
     /// The path to an interface definition file the component exports.
-    #[clap(long = "export", value_name = "NAME=INTERFACE", parse(try_from_str = parse_named_interface))]
```

Comment:
AFAICT the only API differences are here, on how to spell "here's my custom string parser"...